heron/schedulers/src/java/com/twitter/heron/scheduler/ecs/EcsContext.java

```diff
@@ -0,0 +1,60 @@
+// Copyright 2016 Twitter. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.twitter.heron.scheduler.ecs;
+
+import com.twitter.heron.spi.common.Context;
+
+
+/**
+ * Created by ananth on 4/29/17.
+ */
+public class EcsContext extends Context {
+  public static final String PART1 = "version: '2'\n"
+      + "services:\n"
+      + "  container_number:\n"
+      + "    image: ananthgs/onlyheronandubuntu\n";
+  public static final String CMD = "    command: [\"sh\", \"-c\", \"mkdir /s3; cd /s3 ;"
```

Comment:
Removed the hard coding. Created a template.yaml file which will be used instead.